texastribune/donations-app
# Top renew/donate buttons update

#### What's this PR do?

Applies post-drive updates to the top checkout. Switches out the current "Renew" button for a "Renew" and a "Donate" option. Both of these buttons link to a blank, annual Givalike form.
#### Why are we doing this? How does it help us?

Gives people an option for just donating if that's how they'd like to contribute to the Tribune.
#### How should this be manually tested?

Visit the page locally, and confirm that both "Renew" and "Donate" link through to a blank, annual Givalike form. Check it out at all different screen sizes. If you want to check it out in Browserstack and/or different browsers, bonus points.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

Not really.
#### What are the relevant tickets?

Sprint task: https://basecamp.com/1866004/projects/8824386/messages/41871611
#### Next steps?
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?
#### Technical debt note

Same
